netbox/extras/views.py

```diff
@@ -1007,6 +1007,23 @@ def post(self, request, id):
         return redirect(reverse('home'))
 
 
+#
+# Function for returning job data - used in scripts and reports
+#
+
+# Return jobs filtered for job model type (Example: `scriptmodule`)
+def get_jobs(module, job_object, jobtype):
```

Comment:
Can we move this to a method on the Report & Script classes? That would keep the code closer to the models, and obviate needing to pass `jobs` as context under every view.

(If not, the function should at least be moved to `utils.py`.)

Reply:
Yup, definitely, I figured we wanted to reduce the reproduction.  Are you good if I make a Mixin for the classes?